packages/platform/types/bookings/2024-08-13/inputs/location.input.ts

```diff
@@ -51,7 +51,7 @@ export const supportedIntegrations = [
   "roam-video",
   "riverside-video",
   "ping-video",
-  "office365-video",
```

Comment:
This is a breaking change - if someone was passing `office365-video` now that won't be possible anymore. So we need to:
1. Keep `"office365-video"` and `"msteams"` here while hiding `"office365-video"` from input docs.
2. Process them the same in the input.

However, since we also change atoms logic in [transformApiEventTypeForAtom.ts](https://github.com/calcom/cal.com/pull/21255/files#diff-160b5b148862bf20a83a1a27b77be36bfa4339a5972ecb1bff22bdf83323c2cd) wouldn't that mean that peopel with older atoms will have it breaking, because api will only return "msteams" ?

Because of this I would just convert "office365-video" to "msteams" internally and then when returning data.

Reply:
API accepts office365-video -> converts to msteams so code works -> returns office365-video so the output does not change.

As for atoms maybe we need something similar.